engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/DateTest.scala

```diff
@@ -0,0 +1,55 @@
+package org.enso.interpreter.test.semantic
+
+import org.enso.interpreter.test.{InterpreterContext, InterpreterTest}
+
+class DateTest extends InterpreterTest {
+  override def subject: String = "Debug.eval"
+
+  override def specify(implicit
+    interpreterContext: InterpreterContext
+  ): Unit = {
+    "evaluate a date expression" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |
+           |import Standard.Base.Data.Time.Date
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        IO.println (Date.new 2022 04 01)
+           |""".stripMargin
+      eval(code)
+      consumeOut shouldEqual List("2022-04-01")
+    }
+
+    "print out java date" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |polyglot java import java.time.LocalDate
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        IO.println (LocalDate.of 2022 04 01)
+           |""".stripMargin
+      eval(code)
+      consumeOut shouldEqual List("2022-04-01")
+    }
+
+    "compare dates in java" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |polyglot java import java.time.LocalDate
+           |import Standard.Base.Data.Time.Date
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        ensodate = Date.new 2022 04 01
+           |        javadate = LocalDate.of 2022 04 01
+           |        IO.println (javadate.compareTo ensodate)
```

Comment:
I had to find a method that takes `LocalDate` as parameter and pass `ensodate` there - that's the problematic use-case. I thought `LocalDate.compareTo` would do, but it doesn't. The [compareTo](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalDate.html#compareTo(java.time.chrono.ChronoLocalDate)) takes `ChronoLocalDate` - an interface that `LocalDate` implements. That confuses Truffle interop...